docs/swagger.yaml

```
@@ -864,4 +965,43 @@ paths:
       summary: Tehnicki pregled
       tags:
       - vehicle
+  /vehicle/vin/{vin}:
+    get:
+      parameters:
+      - description: Vehicle VIN number
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)